src/main/java/core/basesyntax/FileWork.java

```diff
@@ -1,7 +1,35 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.Arrays;
+
 public class FileWork {
+    public static final String SPECIFIED_CHARACTER = "w";
+    public static final String SEPARATOR = " ";
```

Comment:
```suggestion
    public static final String SEPARATOR = "\\W+";
```
this way you won't need to call replaceAll